src/d3a/resources/Solar_Curve_W_cloudy_normalized.csv

```diff
@@ -0,0 +1,97 @@
+INTERVAL;POWER(W)

```

Comment:
I was expecting it to be kW

Reply:
Okay, how about using the pre-existing CSVs  in codebase?

Reply:
Sorry, don't understand. What do you mean by `pre-existing CSVs` ?
The scaling of the predefined load is a feature requested in the ticket.

Reply:
I meant why not updating the old CSV itself `Solar_Curve_W_cloudy` rather than creating this one, as ATM, they both only been used in `src/d3a/models/strategy/predefined_pv.py`

Reply:
As described in the ticket, its discussion and in the documentation I provided here https://gridsingularity.atlassian.net/wiki/spaces/D3A/pages/3008004097/PV+Workflow+d3a-web+--+d3a+as+of+Jul+2021  We need a way of scaling the predefined PV with the `capacity_kW`. That is why we normalized the predefined profiles so that we only need to multiply `capacity_kW`.

Reply:
Sorry, still haven't understood it
<img width="857" alt="Screen Shot 2021-08-05 at 11 02 26" src="https://user-images.githubusercontent.com/33482427/128323379-ec5dab91-80b9-4a1d-a420-47b602a0b5d8.png">
There your talking about both `capacity_kW` as well as `max_panel_powerW` whereas what I can see in `PVPredefinedStrategy`, no `max_panel_powerW` parameter exist now
![Screen Shot 2021-08-05 at 11 04 30](https://user-images.githubusercontent.com/33482427/128323666-ec6cd2e3-e40e-47cf-b442-cb3b533abd84.png)

Reply:
You looked at the d3a-web paragraph. But you are right, there was no max_panel_power before in the PredefinedPV. `capacity_kW` was added now to scale the predefined profiles (which again are normalized to be between 0 and 1).

Reply:
I just realise that the numbers never reach one. But it was decided that the exiting profiles should be divided by 250.

Reply:
> which again are normalized to be between 0 and 1

Would be nice then to have validator function for validating that the provided CSVs/profiles always have a value between 0 and 1.

Reply:
IMO there is no need. You can not upload such a profile, we only support these three: Sunny, Cloudy, PartiallyCloudy

Reply:
> IMO there is no need. You can not upload such a profile, we only support these three: Sunny, Cloudy, PartiallyCloudy

Okay thanks got it after looking into PR again, BUT still why are you still keeping the profile that were used earlier ?

- Solar_Curve_W_partial.csv
- Solar_Curve_W_sunny.csv
- Solar_Curve_W_cloudy.csv

Reply:
Yeah, I could remove them, but I thought that people might want to use them as example profiles for the PVUserProfileStrategy. If you like me to delete them, will do, but I don't see any harm in leaving them.

Reply:
Okay, leave it then.